src/SFA.DAS.AdminService.Web/Services/AnswerService.cs

```diff
@@ -164,6 +166,39 @@ public async Task<CreateOrganisationStandardCommand> GatherAnswersForOrganisatio
             return command;
         }
 
+        public async Task<WithdrawOrganisationRequest> GatherAnswersForWithdrawOrganisationForApplication(Guid applicationId, string updatedBy)
+        {
+            var application = await _applyApiClient.GetApplication(applicationId);
+            var applicationData = await _qnaApiClient.GetApplicationDataDictionary(application?.ApplicationId ?? Guid.Empty);
+            var organisation = await _apiApiClient.GetOrganisation(application?.OrganisationId ?? Guid.Empty);
+
+            if (application is null || applicationData is null || organisation is null) return null;
+
+            return new WithdrawOrganisationRequest
+            {
+                ApplicationId = application.Id,
+                EndPointAssessorOrganisationId = organisation.EndPointAssessorOrganisationId,
+                WithdrawalDate = (DateTime)applicationData[nameof(ApplicationData.ConfirmedWithdrawalDate)],
+                UpdatedBy = updatedBy
+            };
+        }
+
+        public async Task<WithdrawStandardRequest> GatherAnswersForWithdrawStandardForApplication(Guid applicationId)
+        {
+            var application = await _applyApiClient.GetApplication(applicationId);
+            var applicationData = await _qnaApiClient.GetApplicationDataDictionary(application?.ApplicationId ?? Guid.Empty);
+            var organisation = await _apiApiClient.GetOrganisation(application?.OrganisationId ?? Guid.Empty);
+
+            if (application is null || applicationData is null || organisation is null) return null;
```

Comment:
These lines are a duplicate of lines 171-175. If this is intentional, they can be refactored into a class that returns the three as a DTO. That way we can add some quick tests. (Test coverage is very low in this PR and in the Admin Service in general)